pkg/logutil/context.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logutil
+
+import (
+	"context"
+
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+type ctxLogKeyType struct{}
+
+// CtxLogKey indicates the context key for logger
+// public for test usage.
+var CtxLogKey = ctxLogKeyType{}
```

Comment:
Why we make the `CtxLogKey` public?

Reply:
Let user use the `context.Value(CtxLogKey)` freely.
But it can be private.

Reply:
Fixed.